src/commands/react-native/interfaces.ts

```diff
@@ -67,6 +56,38 @@ export class RNSourcemap {
 
     return splitPath[splitPath.length - 1]
   }
+
+  private getMetadataPayload(
+    cliVersion: string,
+    service: string,
+    version: string,
+    projectPath: string,
+    platform: RNPlatform,
+    build: string
+  ): MultipartValue {
+    const metadata: {[k: string]: any} = {
+      build_number: build,
+      bundle_name: this.bundleName,
+      cli_version: cliVersion,
+      platform,
+      project_path: projectPath,
+      service,
+      type: 'react_native_sourcemap',
+      version,
+    }
+    if (this.gitData !== undefined) {
+      metadata.git_repository_url = this.gitData!.gitRepositoryURL
+      metadata.git_commit_sha = this.gitData!.gitCommitSha
```

Comment:
nitpick: `!` don't do anything here since we already checked that `this.gitData` is not undefined, but they were there before so not a blocker for me.